src/anim/evaluator/anim-data.js

```diff
@@ -18,7 +16,7 @@ class AnimData {
     }
 
     /**
-     * The number of components that make up an element.
+     * Gets the number of components that make up an element.
```

Comment:
AnimData does not have very good explanation at what it contains .. and so I wonder if this should be public at all? Perhaps the properties of the AnimTrack that return AnimData should be private? Or do we expect user to do anything with the data.

Reply:
I've made the AnimCurve and AnimData classes private and hidden the constructor for the AnimTrack class as the goal of this PR was to display the AnimTrack class correctly in the assignAnimation method rather than provide public API for its internals.